packages/api-server/src/methods/modules/eth.ts

```diff
@@ -989,7 +1002,7 @@ export class Eth {
       }
 
       const fromBlockNumber: U64 = await this.blockParameterToBlockNumber(
-        filter.fromBlock || "earliest"
+        filter.fromBlock || "latest"
```

Comment:
not sure if this is in conflict with https://github.com/nervosnetwork/godwoken-web3/pull/300/files/131542bd5cc272279d27760e258fb5fa5de6fc9a#diff-cce75f92de0f01f77430b0195c78d8a547ca11f2b8f818202008dce92a69ebef ?

cc @classicalliu

Reply:
Why change to `latest` block here ?

Reply:
In order to make it compatible, I've changed it to `latest`. Here are Geth and Alchemy declaring that `fromBlock` and `toBlock` should be `latest`:
  - https://github.com/ethereum/go-ethereum/blob/10dc5dce0871bf8c24bac41b04e47c3b9ad2b93e/eth/filters/api.go#L280
  - https://docs.alchemy.com/alchemy/apis/ethereum/eth-newfilter#parameters

Reply:
Have you test this issue https://github.com/nervosnetwork/godwoken-web3/issues/273 ?

It's `fromBlock` should be latest block when `newFilter` , not the latest block when `getFilter`

`earliest` will always lower than id cursor, so the true from is the id cursor, if this be `latest`, `from` will always equals to `to` if no `fromBlock` & `toBlock` provided. And `id` to current latest block will be lost, so it changed from `latest` to `earliest`

Reply:
@classicalliu Thanks for your correction. Yes, you are right. Unlike Geth and other projects that use subscriptions, we implement the filter by querying the database.

I will change it back to "earliest".
Would you mind if I put your response into the code comment?

Reply:
[8e22fd0](https://github.com/nervosnetwork/godwoken-web3/pull/427/commits/8e22fd099896cc0d1da0848971399a9f29037d81) reverted changes. PTAL. @classicalliu @RetricSu

Reply:
> @classicalliu Thanks for your correction. Yes, you are right. Unlike Geth and other projects that use subscriptions, we implement the filter by querying the database. They don't have to "get filter".
> 
> I will change it back to "earliest". Would you mind if I put your response into the code comment?

Of course not